recipes/maxentscan/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: maxentscan
+  version: "1.0"
+source:
+  fn: fordownload.tar.gz
```

Comment:
Hi Johannes,

Sadly, I cannot, as I have no say, nor any relation to the people who wrote the scripts. I am merely trying to make it easily installable using conda :/